netbox/netbox/api.py

```diff
@@ -12,7 +12,10 @@
 
 def is_custom_action(action):
     return action not in {
-        'retrieve', 'list', 'create', 'update', 'partial_update', 'destroy', 'bulk_destroy'
+        # Default actions
+        'retrieve', 'list', 'create', 'update', 'partial_update', 'destroy',
+        # Bulk operations
+        'bulk_update', 'bulk_partial_update', 'bulk_destroy',
     }
```

Comment:
Wonder if it would be better to do something like:

```python
from restframework.schemas.coreapi import is_custom_action as base_is_custom_action

def is_custom_action(action):
    return base_is_custom_action(action) and action not in {
        'bulk_update', 'bulk_partial_update', 'bulk_destroy',
    }
```

So that we are clearly extending the base function, just in case it ever changes?

Reply:
If the built-in operations change, we'll have much, much bigger signs than the schema generation failing. :smile: